dts/bindings/gpio/zephyr,sim-gpio.yaml

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020, Friedt Professional Engineering Services, Inc
+# SPDX-License-Identifier: Apache-2.0
+
+description: GPIO Simulator
```

Comment:
Can you add a little more detail to the description.  Something like 'Native Posix simulated GPIO - See drivers/gpio/Kconfig.sim... for more details'

Reply:
>  Something like 'Native Posix simulated GPIO

I thought it was agreed this is not specific to Native Posix...

Reply:
It is not :)